packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.markup.tests.js

```diff
@@ -139,6 +138,28 @@ QUnit.module('markup', {
         return $element.length && !$element.hasClass('dx-state-invisible');
     }
 }, () => {
+    QUnit.test('shouldnt contain <a> node tag when url is passed as attribute (T1237378)', function(assert) {
+        const items = [
```

Comment:
[minor] we dont need const here, neither for reusability nor for readability, lets inline it

Reply:
[d45533d](https://github.com/DevExpress/DevExtreme/pull/27647/commits/d45533db3d0fdfffa3fcf4d8a704612c6f429a4e)